assets/resumes/README.md

```diff
@@ -0,0 +1,46 @@
+# Resume Examples
```

Comment:
maybe these can be moved to a particular place in json? OR .resume templates redirects them to THIS readme

i guess u can delete all the PDFs in /assets

Reply:
Well, the pdfs in /assets are just templates from Overleaf, so we probably will not use these for examples.

I'll delete the pdfs in the next commit.